Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -65,9 +65,13 @@ export class JavaFilesExtractor {
 
     private async tarExtract(file, destinationFolder) {
```

Comment:
We should be using `extractTar` from vsts-task-tool-lib for this.  I see the implementations are the same.  If this broke, we should push the fix upstream

Reply:
The extractTar method does not allow for specifying which destination folder should be used.  Using the extractTar method would require a change to both the vsts-tasks code base as well as the vsts-tasks-tool-lib code. In order to get this fix out sooner, I will be porting my current fix to the previous release and will abandon this PR in favor of making changes to both repos.